src/Services/ApmCollectorService.php

```diff
@@ -75,4 +77,13 @@ public function addCollector(string $collector_class): void
             $this->app->make($collector_class)
         );
     }
+
+    public function captureThrowable(Throwable $thrown, array $context = [], ?Transaction $parent = null)
+    {
+        if ($this->is_agent_disabled) {
+            return;
+        }
+
+        $this->app->make(Agent::class)->captureThrowable($thrown, $context, $parent);
```

Comment:
Not sure how Laravel's service container works, but I hope this only creates one instance of the Agent that will be reused on every call to `captureThrowable`.

Reply:
Correct, there is only one Agent instance because the AG Agent is registered as a singleton in the ServiceProvider.